src/util.js

```diff
@@ -69,12 +69,18 @@ export function podmanAction(name, args, root) {
     return varlink.call(address, "io.podman." + name, args, root);
 }
 
-export function monitor(name, args, callback, root) {
+export function monitor(name, args, callback, error_callback, root) {
```

Comment:
Let's please call this `on_close`, as that's what it does -- it does not receive general errors. Please also add a comment what argument it receives (`root`), and also what `root` actually means.